File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    let n=nums.length;
    if(n==1) return 1;
    let j=1;
    for(let i=1;i<n;i++){
        if(nums[i]!=nums[j-1]) {
            nums[j]=nums[i];
            j++;
        }
    }

    return j;
};

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        if len(nums)==1:return 1;
        i=1;
        j=1;
        for i in range(1,len(nums)):
            if nums[i]!=nums[j-1]:
                nums[j]=nums[i];
                j=j+1;
        return j;
        